src/config/dynamic-theme-fixes.config

```diff
@@ -13294,9 +13294,15 @@ img.act-icon-dark-gray
 [src="/chrome/static/images/google-translate/screen-english.png"]
 
 CSS
-.RNNXgb,
+.minidiv .sfbg,
+#searchform.minidiv,
+.minidiv .emcav .RNNXgb .S6VXfe,
+.CvDJxb .RNNXgb,
 .aajZCb {
-    box-shadow: 0 0 2px 0 ${rgba(0,0,0,0.16)}, 0 0 0 1px ${rgba(0,0,0,0.08)} !important;
+    box-shadow:0 1px 6px 0 rgba(0,0,0,.28) !important;
```

Comment:
I don't think this is a good idea. The lighter box-shadow makes it more notice-able for these elements. Thereby making shadows darker than the original element makes it the shadow itself fade in the background and not distinguishable. Also please preserve the `${}` behavior.

Reply:
Well, how about stronger shadows? 2px blur radius and 3px spread along with using the `${}` (sorry I didn't do that the first time) makes it much more visible even on darker backgrounds:
`box-shadow:0 1px 2px 3px ${rgba(255,255,255,.28)} !important;`
Perhaps we could even try increasing opacity if this doesn't suffice?

And I think having a light colored shadow somewhat deteriorates the experience, as it no longer has the appeal of a normal shadow.